doc/user/content/integrations/cdc-mysql.md

```diff
@@ -113,8 +113,6 @@ CREATE SOURCE kafka_repl
     ENVELOPE DEBEZIUM;
 ```
 
-If log compaction is enabled for your Debezium topic, you must use `ENVELOPE DEBEZIUM UPSERT`. This will require **more memory** in Materialize, as it needs to track state proportional to the number of unique primary keys in the changing data.
```

Comment:
> This will require **more memory** in Materialize, as it needs to track state proportional to the number of unique primary keys in the changing data.

It feels like mentioning that `ENVELOPE DEBEZIUM` will require memory proportional to the number of unique keys is still relevant?

Reply:
I was torn on this one, and the other one I removed. It's an implementation detail that we will (hopefully) address at some point. And it might now be clear _where_ that memory is required. We know that it's in the `storaged` pod, but I don't know how much we want to expose those concepts to the user.

Reply:
Let's just roll without it, then.

Reply:
Oh, we can add it to the "sizing a source" section, then! We already call out that you'll need to size up upsert envelope sources if you have lots of unique keys. It'd be real natural, IMO, to add "and debezium envelope sources" there.

https://materialize.com/docs/unstable/sql/create-source/#sizing-a-source